Add missing `make config-reset` to "useful make commands" in developer-workflow (#537)
mattermost/mattermost-developer-documentation: site/content/contribute/server/developer-workflow.md

* `make stop-server` stops only the server.
* `make clean-docker` stops and removes your Docker images and is a good way to wipe your database.
* `make clean` cleans your local environment of temporary files.

> * `make config-reset` resets your local config files like `config/config.json`

Reviewer: ```suggestion
* `make config-reset` resets the `config/config.json` file to the default.
```
